styles/styles.css

```diff
@@ -1094,3 +1101,7 @@ main .section.blue a.button.secondary::after {
 .tag-green {
   background-color: var(--green-tag);
 }
+
+.products-container h4 {
```

Comment:
If these are product specific, should they be part of the product block css ?

Reply:
I don't think so since the headings are outside of the block but would be happy to change if you disagree @rhudea